### ISSUE TITLE
Support for AV1 format

### ISSUE THREAD

**Issue by damian101** [zeroepoch/plotbitrate#25]
I had to set up my operating system again, Arch Linux, because my SSD died, and now plotbitrate doesn't seem to support AV1 video anymore, I always get an empty graph, 0 Mbit/s.
I assume this might be an ffprobe issue, but posting here regardless as I don't have the knowledge to investigate further.

**Comment by zeroepoch** [comment]
@damian101, any chance you could share a small clip of the video you're having problems with? I went looking online for a small sample clip encoded using AV1 and found https://github.com/SPBTV/video_av1_samples/blob/master/spbtv_sample_bipbop_av1_960x540_25fps.mp4

```
Input #0, mov,mp4,m4a,3gp,3g2,mj2, from 'spbtv_sample_bipbop_av1_960x540_25fps.mp4':
  Metadata:
    major_brand     : mp42
    minor_version   : 0
    compatible_brands: avc1isommp42avc1
    creation_time   : 2018-10-05T14:40:45.000000Z
  Duration: 00:00:15.04, start: 0.000000, bitrate: 130 kb/s
  Stream #0:0[0x1](und): Audio: aac (LC) (mp4a / 0x6134706D), 32000 Hz, mono, fltp, 8 kb/s (default)
    Metadata:
      handler_name    : snd
      vendor_id       : [0][0][0][0]
  Stream #0:1[0x2](und): Video: av1 (Main) (av01 / 0x31307661), yuv420p(tv), 960x540, 116 kb/s, 25 fps, 25 tbr, 90k tbn (default)
    Metadata:
      handler_name    : vid
      vendor_id       : [0][0][0][0]
```

This clip appears to work fine on my Fedora 36 system after I updated some matplotlib stuff (unrelated to this bug).

![out](https://user-images.githubusercontent.com/1310250/185768790-e79052a7-fdaf-4484-a541-995b338017f2.png)

**Comment by zeroepoch** [comment]
I think on my system it's using `libdav1d` for the decoding.

**Comment by damian101** [comment]
> @damian101, any chance you could share a small clip of the video you're having problems with? I went looking online for a small sample clip encoded using AV1 and found https://github.com/SPBTV/video_av1_samples/blob/master/spbtv_sample_bipbop_av1_960x540_25fps.mp4
> 
> ```
> Input #0, mov,mp4,m4a,3gp,3g2,mj2, from 'spbtv_sample_bipbop_av1_960x540_25fps.mp4':
>   Metadata:
>     major_brand     : mp42
>     minor_version   : 0
>     compatible_brands: avc1isommp42avc1
>     creation_time   : 2018-10-05T14:40:45.000000Z
>   Duration: 00:00:15.04, start: 0.000000, bitrate: 130 kb/s
>   Stream #0:0[0x1](und): Audio: aac (LC) (mp4a / 0x6134706D), 32000 Hz, mono, fltp, 8 kb/s (default)
>     Metadata:
>       handler_name    : snd
>       vendor_id       : [0][0][0][0]
>   Stream #0:1[0x2](und): Video: av1 (Main) (av01 / 0x31307661), yuv420p(tv), 960x540, 116 kb/s, 25 fps, 25 tbr, 90k tbn (default)
>     Metadata:
>       handler_name    : vid
>       vendor_id       : [0][0][0][0]
> ```
> 
> This clip appears to work fine on my Fedora 36 system after I updated some matplotlib stuff (unrelated to this bug).
> 
> ![out](https://user-images.githubusercontent.com/1310250/185768790-e79052a7-fdaf-4484-a541-995b338017f2.png)

With the same sample I get an empty graph... so it's something about my system, not the specific sample...

**Comment by damian101** [comment]
So... Now it works again, was still broken last week. The one thing I changed on my system was uncommented the locale I had set in `/etc/locale.gen` and executed `locale-gen`; for some reason all were commented out in there. Which caused mkvmerge to break, which is how I noticed... However, a week ago mkvmverge worked while plotbitrate did not with AV1, so might be completely unrelated.
Anyway, closing this for now...
